draft-trammell-quic-spin.md

```diff
@@ -262,7 +262,7 @@ bit, as described in {{usage}}.
 
 Since it is possible to measure handshake RTT without a spin bit (see
 {{handshake}}), it is sufficient to include the spin bit in the short
-packet header. This proposal suggests to use the second most significant bit
+packet header. This proposal suggests to use the fourth most significant bit
 (0x40) of the first octet in the short header for the spin bit.
```

Comment:
Not sure if 0x40 matches the header figure.

Reply:
yeah, this should be 0x10, will fix after merging.